src/main/java/com/wsclient/service/RestClientService.java

```diff
@@ -46,6 +46,7 @@ public ServiceData post(ServiceData serviceData) throws JsonParseException, Json
 		HttpHeaders headers = new HttpHeaders();
 		headers.setContentType(MediaType.APPLICATION_JSON);
 		HttpEntity<String> httpEntity = new HttpEntity<>(serviceData.getRequestBody(), headers);
+		CustomFactory customFactory = new CustomFactory();
```

Comment:
This doesn't seems to be used anywhere. Please remove this line if you find the same appropriate.